requirements.txt

```diff
@@ -10,3 +10,4 @@ uncertainties
 loguru
 nestle>=0.2.0
```

Comment:
should we discuss having a new requirement more broadly?  If this is only needed for a subset of tasks, should it be optional?

Reply:
I think this was actually adding `joblib`?

Reply:
In particular, I wonder about how using `joblib` compares to the use of `concurrent.futures`?  I see some discussion online.  It seems like we might want to stick with one library for that functionality.

Reply:
I will try `concurrent.futures`.